### DISCUSSION TITLE
Change the outputted DLLs to include the compiled with .NET version?

### DISCUSSION THREAD

**Discussion by Protiguous** [AdamWhiteHat/BigDecimal#34] [Ideas Or Feature Requests]
@AdamWhiteHat, I believe I have found the way to change the outputted DLL's names to include the compiled-with .NET version.
It still needs some testing to verify it actually works.

So, did I understand you correctly?
You want, “**ExtendedNumerics.BigDecimal-net45.dll**” then?

**Comment by AdamWhiteHat** [comment]
Naw, I meant like how with some libraries, the version numb 3.5 was compiled to use framework 3.5, and version 4.8 was compiled to use framework version 4.8.

But that ship has long since sailed.

**Reply by Protiguous** [reply]
Ah, okay. I'll add the file.cs to do it to a gist or something then.
